src/components/molecules/FilterMenu/FilterMenuFooter/FilterMenuFooter.tsx

```diff
@@ -0,0 +1,17 @@
+import {StyledOkButton, StyledResetButton, StyledSpace} from './FilterMenuFooter.styled';
+
+type Props = {
+  onReset: () => void;
+  onOk: () => void;
+};
+
+const FilterMenuFooter: React.FC<Props> = ({onReset, onOk}) => {
```

Comment:
Use here `props` argument and destructure it below

Reply:
done